vulnfeeds/cpp/main.go

```diff
@@ -131,6 +131,41 @@ func GitVersionsToCommit(CVE string, versions cves.VersionInfo, repos []string,
 	return v, nil
 }
 
+func refAcceptable(ref cves.CVEReferenceData, tagDenyList []string) (acceptable bool) {
+	for _, deniedTag := range tagDenyList {
+		if slices.Contains(ref.Tags, deniedTag) {
+			return false
+		}
+	}
+	return true
+}
+
+// Examines the CVE references for a CVE's CPE and derives repos for it.
+func ReposForCPE(CVE string, cache VendorProductToRepoMap, vp VendorProduct, refs []cves.CVEReferenceData, tagDenyList []string) (repos []string) {
+	// This currently only gets called for cache misses, but make it not rely on that assumption.
+	cachedRepos, ok := cache[vp]
+	if ok {
```

Comment:
nit: prefer

```golang

if cachedRepos, ok := cache[vp]; ok {
  return cachedRepos
}
```

It's shorter and scopes `cachedRepos` better

Reply:
Fixed in [b2298f2](https://github.com/google/osv.dev/pull/1164/commits/b2298f296852fd5a931bb67374faaa0c733701fa)